app/src/main/java/com/example/pokedex/ui/PodexListAdapter.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.pokedex.ui
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.example.pokedex.databinding.ViewHolderPokedexListItemBinding
+import com.example.pokedex.domain.ListItem
```

Comment:
so the reason I did this is because my domain passes the ListItem class to the viewState as List<ListItem> because everything in the API is arranged by ListItems (ie a name and a url). I was planning on using both of those, but may change it. If I were to use it like that, what would be the best way to do that? recreate the same class again?

Reply:
Done.